### DISCUSSION TITLE
Searching for a solution to add or remove an existing tag.

### DISCUSSION THREAD

**Discussion by Icqtoni** [benclaussen/NetboxPS#47] [Q&A]
Unfortunately, I don't see any way to remove or add an existing tag to an IP address, for example.
Sorry in am new here, can you please help me out?

$nbIp = Get-NetboxIPAMAddress -Address '172.172.172.172/32'
nbIp.tags =  $nbIp.tags | ? { $_.Name -ne 'offline' }
Set-NetboxIPAMAddress -Id $nbIp.id -??? $nbIp.tags

Thanks in advance

**Comment by alagoutte** [comment]
Hi,

Do you have found the solution ? you want to change tag for IPAMAddress ?

**Comment by Icqtoni** [comment]
Hi,

No, I haven't found a solution.
Yes, I want to change the IPAMAddress tag.

**Comment by WagoL** [comment]
Late to party but this works.
`$ip | Set-NetboxIPAMAddress -Status 'dhcp' -Tags @()`
